boards/vngiotlab/nrf51_vbluno51/doc/index.rst

```diff
@@ -138,8 +138,8 @@ Testing the VBLUno51 with Zephyr: buttons, LEDs, UART, BLE
  * :ref:`hello_world`
  * :zephyr:code-sample:`blinky`
  * :zephyr:code-sample:`button`
- * :ref:`bluetooth-beacon-sample`
- * :ref:`peripheral_hr`
+ * :zephyr:code-sample:`bluetooth_beacon`
+ * :zephyr:code-sample:`ble_peripheral_hr`
```

Comment:
why is one prefixed with `bluetooth_` and the other with `ble_`? Don't think the beacon sample runs on a classic stack.
I would prefer `bluetooth_classic_xx` for classic samples, and the rest is assumed to be LE. 
What do you think?

Reply:
There's a few more with the same inconsistency. Also some that is prefixed with `_`, and even one that is `bluetooth_bt` :D 

Are we planning on using explicit tags, or should we attempt to shorten them at bit? e.g.
```
bluetooth_beacon => bt_beacon
bluetooth_direction_finding_central => bt_df_central
```

Reply:
I would prefer long names for the documentation references. Shortening the filenames/path should be ok. Although i hope people have shell completion

Reply:
The simplest route might be to just use the same IDs as the folder names?
Related: is it time to introduce some subfolders underneath `samples/bluetooth`? It's starting to get really packed in there :)

Reply:
> The simplest route might be to just use the same IDs as the folder names? Related: is it time to introduce some subfolders underneath `samples/bluetooth`? It's starting to get really packed in there :)

https://github.com/zephyrproject-rtos/zephyr/discussions/64443 :)

Reply:
> The simplest route might be to just use the same IDs as the folder names? Related: is it time to introduce some subfolders underneath `samples/bluetooth`? It's starting to get really packed in there :)

Sure, so we can have `bluetooth` prefix for all BT samples, and then, when we do the above move, we can have them like `bluetooth_audio_xxx`, `bluetooth_mesh_xxx`